src/ToDo/Business/TaskListService.cs

```diff
@@ -4,11 +4,13 @@ namespace ToDo.Business;
 public class TaskListService : ITaskListService
 {
 	private readonly ITaskListEndpoint _client;
+	private readonly ITaskService _taskSvc;
```

Comment:
I think it would be better to move the GetTasksAsync method into the ITaskService, thus avoiding the interdependency between these two services

Reply:
Indeed I tried the other way (moving `GetByFilterAsync` and `GetAllAsync` from the `TaskEP` to the `TaskListEP`) but it was weird. Much better this way.